clock_exercise/DateSettingPage.qml

```diff
@@ -0,0 +1,116 @@
+import QtQuick 2.0
+
+Rectangle {
+    id: dataSettingPage
+
+    signal back
+
+    color: "#151B2E"
+
+    BackButton {
+        x: 28
+        y: 20
+        onBack: dataSettingPage.back()
+    }
+
+    TitleOfThePage {
+        y: 40
+        anchors.horizontalCenter: parent.horizontalCenter
+        text: qsTr("Set date")
+    }
+
+    LineUnderTitleOfThePage {
+        y: 115
+        anchors.horizontalCenter: parent.horizontalCenter
+    }
+
+    Row {
+        y: 140
+        anchors.horizontalCenter: parent.horizontalCenter
+
+        DateNumberField {
+            id: day
+
+            numberFontSize: 120
+            value: clockManager.dayOfDate
+            text: clockManager.dayOfDateName
+            onIncrease: {
+                clockManager.addToDate(1, 0, 0)
+            }
+            onDecrease: {
+                clockManager.addToDate(-1, 0, 0)
+            }
+            onSetUpDownModality: {
+                day.upDownModality = true
+                month.upDownModality = false
+                year.upDownModality = false
```

Comment:
Come dicevo, l'idea è di impostare solo il giorno, mai il mese e l'anno. In questa ottica, tutto questo codice può andar via perché è sempre selezionato il giorno e basta.